docs/concepts/trading-on-vega/market-protections.md

```diff
@@ -112,27 +112,27 @@ If a market enters into a liquidity auction and never again attracts enough liqu
 ## Distressed traders
 If a trader's available margin on a market is below the closeout level and cannot be replenished, that trader is considered to be distressed.
 
-A distressed trader has all their open orders on that market cancelled. The network will then recalculate the margin requirement on the trader's remaining open position. If they then have sufficient collateral, they are no longer considered a distressed trader. 
+A distressed trader using cross-margining has all their open orders on that market cancelled. The network will then recalculate the margin requirement on the trader's remaining open position. If they then have enough collateral, they are no longer considered a distressed trader. 
 
-However, if the trader does not have sufficient collateral, they are added to a list of traders that will then undergo position resolution to close out their positions.
+However, if the trader does not have sufficient collateral, their position is closed out.
 
 ### Closeouts
-When a participant does not have enough collateral to hold their open positions, the protocol will automatically trigger a closeout.
+When a participant does not have enough collateral to hold their open positions, the protocol will automatically trigger a closeout. The closeout process is a last resort for a position. 
 
-The closeout process is a last resort for a position. If a trader's deployed margin on the market is insufficient to cover a mark to market settlement liability, first Vega will search the trader's available balance of the settlement asset. If this search is unable to cover the full liability, the trader will be considered distressed and undergo position resolution. Any margin balance remaining after closeout is transferred to the market's insurance pool.
+If a trader's deployed margin on the market is insufficient to cover their mark to market settlement liability, first Vega will search the trader's available balance of the settlement asset. If the trader's full liability still can't be covered, their position is transferred to the network. Any margin balance remaining after their closeout is transferred to the market's insurance pool.
```

Comment:
```suggestion
How a trader's position is treated in a situation where they have insufficient deployed margin to cover their mark to market settlement liability depends on the margin mode they're using. 

If a trader using isolated margin doesn't have enough deployed margin on the market, then the position is closed out. Any margin balance remaining after their closeout is transferred to the market's insurance pool.

If a trader using cross margining doesn't have enough deployed margin on the market, first Vega will search the trader's available balance of the settlement asset. If the trader's full liability still can't be covered, their position is transferred to the network. Any margin balance remaining after their closeout is transferred to the market's insurance pool.
```

Reply:
@witgaw Could you sense check my edit here? The one thing I couldn't confidently confirm is what happens to the bits of margin leftover after a closeout in isolated margin.